examples/hive/k8s/deploy/metastore.yaml

```diff
@@ -0,0 +1,40 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: hivemetastore
+spec:
+  type: "NodePort"
+  selector:
+    app: hivemetastore
+  ports:
+    - port: 9083
+      targetPort: 9083
+      name: metastore
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hivemetastore
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hivemetastore
+  template:
+    metadata:
+      labels:
+        app: hivemetastore
+    spec:
+      containers:
+        - name: hivemetastore-container
+          image: ${HIVEMETASTORE_IMAGE}
+          imagePullPolicy: "Always"
```

Comment:
Change pull policy, it fails in minikube because it tries pulling the image. **IfNotPresent** should make the trick